doc/api/animation_api.rst

```diff
@@ -60,8 +60,8 @@ general gist is to take an existing bit map (in our case a mostly
 rasterized figure) and then 'blit' one more artist on top.  Thus, by
 managing a saved 'clean' bitmap, we can only re-draw the few artists
 that are changing at each frame and possibly save significant amounts of
-time.  When using blitting (by passing ``blit=True``) the core loop of
-`FuncAnimation` gets a bit more complicated ::
+time.  When using blitting (by passing ``blit=True``), the core loop of
+`FuncAnimation` gets a bit more complicated. ::
```

Comment:
This should be a colon, if anything here.

Reply:
what should be a colon?

Reply:
The period.  The sentence is implicitly carried on in the code below.  I'd actually just leave w/o the period or colon, but a period is incorrect.

Reply:
Please refrain from doing copy-editting via github comment.

Reply:
... its a copy-edit PR.  How would you like the copy editing corrected?  Just force-push?